subsys/bluetooth/host/hci_core.c

```diff
@@ -2210,6 +2252,175 @@ static void role_change(struct net_buf *buf)
 #endif /* CONFIG_BT_BREDR */
 
 #if defined(CONFIG_BT_SMP)
+static int addr_res_enable(u8_t enable)
+{
+	struct net_buf *buf;
+
+	BT_DBG("%s", enable ? "enabled" : "disabled");
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_SET_ADDR_RES_ENABLE, 1);
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	net_buf_add_u8(buf, enable);
+
+	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_SET_ADDR_RES_ENABLE,
+				    buf, NULL);
+}
+
+static int hci_irk_add(const bt_addr_le_t *addr, u8_t val[16])
```

Comment:
should this be hci_id_add() ?

Reply:
Why? "ID" for me means identity, and that's something I'd associate mainly with the Identity Address. The parameter that's given to this function is an Identity Resolving Key, hence I'd rather call it "irk" than "id".

Reply:
The command is "Add device to RL", which includes its identity address and also *both* its IRKs (local and remote). I think using "irk" here is misleading, I'd rather use id_add(), peer_add() or dev_add()

Reply:
Right. That makes sense. I'll change it to id. Apologies for not thinking this through initially.